developer/general/gsod.md

```diff
@@ -8,101 +8,81 @@ permalink: /gsod/
+From network-level to software-level protections, as well as protections against firmware and hardware attacks, Qubes OS is trying to protect the user from the most significant attacks they encounter so that they can get their work done, safely.
 
-**Brief explanation**: What is the project?
+#### The Project
 
-**Expected results**: What is the expected result in the timeframe given?
+The goal of the project is to improve the documentation related to Disposable VMs and installing Qubes OS on Virtual Machines.
 
-**Knowledge prerequisite**: Pre-requisites for working on the project. What knowledge or resources are needed? If applicable, links to more information or discussions.
+The current Disposable VMs documentation is scarce, inconsistent in places and is scattered across multiple pages, sometimes hard to find. This project involves consolidating it into one or few easy to find pages, covering all related subjects. It should be written in way easy to follow and understand, clearly separating basic use cases, advanced ones and internal details. The terminology should also be used consistently.
 
-**Mentor**: Name and email address.
-```
-### Offline documentation
+Additionally, Qubes OS is missing an installation guide for virtual machines. Users are installing an outdated and unsupported version of Qubes OS (3.2) instead of the supported version. There is unofficial existing installation guide for Qubes OS on a virtual box but it is misleading and lacks documentation. Usually, users face some errors and bugs while installing Qubes OS on a virtual machine. This project also involves writing an installation guide that details how to install Qubes on virtual machines, and how to troubleshoot any issues that may arise. 
 
-**Project**: Offline documentation
+#### Project’s scope
 
-**Brief explanation**: Qubes OS has thorough documentation on the project website, however a user may find it more convenient to view documentation - especially for troubleshooting network issues -- offline on their Qubes machine. This will improve usability for new users and better support users if they need to troubleshoot anything.
+The technical writer will:
 
-**Expected results**: 
+    Review existing Disposable VM documentation
 
- - Review [past discussions on the issue](https://github.com/QubesOS/qubes-issues/issues/1019)
- - Recommend workflow and platform for displaying offline documentation
- - Test workflow and platform to ensure usability and functionality
+    Propose new documentation layout, including split between pages
 
-**Knowledge prerequisite**: 
+    Propose updated and clarified content
 
- - [Markdown][markdown]
+    Provide a new option of installation guide for users working on virtual machines. 
 
-**Mentor**: [Marek Marczykowski-Górecki][team]
+    Review existing problems and provide solutions to them. 
 
-### Create guide on firstboot for new users
+    Giving a warning for using outdated versions of Qubes.
```

Comment:
This part should be rendered as a list, not as code. Let's replace the indents with hyphens.